src/main/java/de/teamlapen/vampirism/command/test/SkillCommand.java

```diff
@@ -53,7 +54,7 @@ private static int skill(CommandSource commandSource, ServerPlayerEntity asPlaye
             (factionPlayer.getSkillHandler()).resetSkills();
             return 0;
         }
-        ISkill skill = ModRegistries.SKILLS.getValue(new ResourceLocation(type));
+        ISkill skill = ModRegistries.SKILLS.getValue(new ResourceLocation(REFERENCE.MODID, type));
```

Comment:
Compatibility for addons could be done like vanilla does.
If no resource domain is given, vampirism is used otherwise the given one is used